prime-router/src/test/kotlin/tokens/TestDefaultJwt.kt

```diff
@@ -0,0 +1,40 @@
+package gov.cdc.prime.router.tokens
+
+import com.okta.jwt.Jwt
+import java.time.Instant
+import java.util.Collections
+import kotlin.collections.LinkedHashMap
+
+class TestDefaultJwt(
```

Comment:
This class is a "mock" of Okta's `DefaultJwt` class.
We couldn't use the "real" one because it is deep in the Okta package, in the `impl` package, which can only be used as a runtime dependency. 
it says in the documentation:
"Classes in this `impl` implementation package may change in NON backward compatible way, and should ONLY be used as a "runtime" dependency."
We couldn't import this package into the test.
So, we made a fake one that mimics what the real Okta Jwt does